locales/en.json

```diff
@@ -255,6 +255,7 @@
     "mail_domain_unknown": "Unknown mail address domain '{domain:s}'",
     "mail_forward_remove_failed": "Unable to remove mail forward '{mail:s}'",
     "mailbox_used_space_dovecot_down": "Dovecot mailbox service need to be up, if you want to get mailbox used space",
+    "mail_unavailable": "Mail adress unavailable",
```

Comment:
I think this message could be improved, e.g. by clarifying that those addresses are reserved and automatically assigned to the first user created

Reply:
done.